src/containers/common/ClassroomManagerContainer.jsx

```diff
@@ -40,6 +36,15 @@ class ClassroomManagerContainer extends React.Component {
     this.setState({ toast: { message: null, status: null } });
   }
 
+  deleteClassroom(id) {
+    // TODO: Add alert to ask if user is sure
+    Actions.deleteClassroom(id).then(() => {
+      // TODO: For API optimization, do we want to instead manually remove the classroom
+      // out of local app state instead of making another API call
```

Comment:
API optimisation: 👍 

That said, I wouldn't be _too_ worried about optimising the Create/Edit/Delete Classroom features since they wouldn't be called too frequently in a short period of time. Refreshing the list of Classrooms via API GET seems fine for now.

I'm trying to think of cases where update-local-state-instead-of-GET-from-API optimisation would be incredibly important, though... when adding Assignments? When Teachers start having so many Classrooms/Students/Assignments that individual GET calls become humongo?